patroni/postgresql/__init__.py

```diff
@@ -101,6 +102,8 @@ def __init__(self, config):
         self._state_entry_timestamp = None
 
         self._cluster_info_state = {}
+        self._has_permanent_logical_slots = True
+        self._enforce_hot_standby_feedback = False
```

Comment:
1. What is the rationale for setting `_has_permanent_logical_slots` to `True` by default ?
2. Does not this  imply `self._enforce_hot_standby_feedback` must also be `True` ?

Reply:
The `_has_permanent_logical_slots` only affects the [cluster_info_query](https://github.com/zalando/patroni/pull/1820/files#diff-72030eab48ce77175c998a2fd1ee9e84fe950e132b66972aa72496532a3923e5R156-R163).
When the Postgresql object is created we don't have the cluster view from DCS yet. Pulling a bit more additional information from postgres doesn't hurt, but will help us to cover some corner-cases.

No, `_enforce_hot_standby_feedback` should not be originally set to True. In case if there are some bugs in the code it could happen that hot_standby_feedback would be enabled by default and there will be no way of disabling it.